Dockerfile

```diff
@@ -6,4 +6,5 @@ FROM eclipse-temurin:17-jre-alpine
 COPY --from=build /src/target/modules /app/modules
 COPY --from=build /src/target/storm.jar /app/storm.jar
 COPY --from=build /src/webroot /webroot
+COPY --from=build /src/etc/storm/config/config.json /etc/storm/config/config.json
```

Comment:
Wouldn't it be better to have config.json file stored in /src/config but copied to /etc/storm/config folde? So we don't need to expose a linux specifik folder structure in our source code repo?

Reply:
Researched this problem a bit for issue #18 where we needed different folders depending on when running locally or in a docker container.
We could have several paths to try if the first one fails. Or we could override the value with an environment variable that can be set both in our IDE and as a configuration in Dockerfile.